src/app/spaces/space.service.ts

```diff
@@ -6,6 +6,7 @@ import { Observable } from 'rxjs';
 
 import { WIT_API_URL } from '../api/wit-api';
 import { Space } from '../models/space';
+import { query } from '@angular/core/src/animation/dsl';
```

Comment:
not needed?

Reply:
Argh auto import on writing text. Sorry and thanks for the catch. Should be gone now.